apps/wallet-mobile/src/features/Swap/useCases/StartSwapScreen/ListOrders/ListOrders.tsx

```diff
@@ -1,48 +1,48 @@
 import React, {useState} from 'react'
 import {ScrollView, StyleSheet, View} from 'react-native'
+import {SafeAreaView} from 'react-native-safe-area-context'
 
 import {Boundary} from '../../../../../components'
+import {useSearchOnNavBar} from '../../../../../Search/SearchContext'
 import {COLORS} from '../../../../../theme'
 import {ButtonGroup} from '../../../common/ButtonGroup/ButtonGroup'
 import {useStrings} from '../../../common/strings'
-import {ClosedOrders} from './ClosedOrders'
-import {OpenOrders} from './OpenOrders'
-
-type Item = {
-  label: string
-  value: string
-}
-
-type SwapOrder = {
-  label: React.ReactNode
-  mainInfo: Item[]
-  hiddenInfo: Item[]
-  buttonAction: () => void
-  buttonText?: string
-}
-
-export type OpenOrderListType = SwapOrder[]
+import {CompletedOrders, CompletedOrdersSkeleton} from './CompletedOrders'
+import {OpenOrders, OpenOrdersSkeleton} from './OpenOrders'
 
 export const ListOrders = () => {
+  const [orderStatusIndex, setOrderStatusIndex] = useState<number>(0)
+
   const strings = useStrings()
 
-  const [orderStatusIndex, setOrderStatusIndex] = useState<number>(0)
-  // TODO: @SorinC6: is it completed or closed orders?
   const orderStatusLabels = [strings.openOrders, strings.completedOrders]
   const handleSelectOrderStatus = (index: number) => {
     setOrderStatusIndex(index)
   }
 
+  useSearchOnNavBar({
+    placeholder: strings.searchTokens,
+    title: strings.swapTitle,
+  })
+
   return (
-    <View style={styles.container}>
-      <ScrollView style={styles.keyboard}>
+    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.container}>
```

Comment:
@banklesss is ListOrders a Screen? do we need SafeAreView here? I thought it was wrapped within another screen.

Reply:
yes you are right! thanks